src/fhir/PlanDefinition_cove.json

```diff
@@ -0,0 +1,438 @@
+    {
+      "resourceType": "ActivityDefinition",
+      "id": "Day371Msg0",
+      "name": "Day 371 Message 0",
+      "status": "active",
+      "kind": "CommunicationRequest",
+      "timingTiming": {
+        "repeat": {
+          "frequency": 1,
+          "period": 371,
+          "periodUnit": "d",
+          "timeOfDay": [
+            "15:33:48"
+          ]
+        }
+      },
+      "dynamicValue": [
+        {
+          "path": "payload.contentString",
+          "expression": {
+            "language": "text/cql",
+            "expression": "Can you believe a year has gone by and I'm sending my last message from this program? I have truly enjoyed being in touch over the past year, {name} and remember that resources are always out there at wefacethefight.org. Know that I’m rooting for you! 💪"
+          }
+        }
+      ]
+    },
+    {
+      "resourceType": "ActivityDefinition",
+      "id": "Day371Msg1",
+      "name": "Day 371 Message 1",
+      "status": "active",
+      "kind": "CommunicationRequest",
+      "timingTiming": {
+        "repeat": {
+          "frequency": 1,
+          "period": 371,
+          "periodUnit": "d",
+          "timeOfDay": [
+            "15:34:20"
+          ]
+        }
+      },
+      "dynamicValue": [
+        {
+          "path": "payload.contentString",
+          "expression": {
+            "language": "text/cql",
+            "expression": "Your final research survey should be headed your way in the next few days - we would appreciate your candid feedback about this program on the survey. -Riley"
+          }
+        }
+      ]
+    },
+    {
+      "resourceType": "ActivityDefinition",
+      "id": "NewYearsMsg",
+      "name": "New Years Message",
+      "status": "active",
+      "kind": "CommunicationRequest",
+      "timingTiming": {
+        "event": [
+          "2025-01-01T17:21:16.000Z"
+        ]
+      },
+      "dynamicValue": [
+        {
+          "path": "payload.contentString",
+          "expression": {
+            "language": "text/cql",
+            "expression": "Sending you warm wishes for a bright year ahead! 🌟❄️🌟❄️ I am grateful to be in touch with you and hope this new year will be a good one, even as the seasons change -Riley"
+          }
+        }
+      ]
+    },
+    {
+      "resourceType": "ActivityDefinition",
+      "id": "VeteransDayMsg",
+      "name": "Veterans Day Message",
+      "status": "active",
+      "kind": "CommunicationRequest",
+      "timingTiming": {
+        "event": [
+          "2024-11-11T17:25:47.000Z"
+        ]
+      },
+      "dynamicValue": [
+        {
+          "path": "payload.contentString",
+          "expression": {
+            "language": "text/cql",
+            "expression": "Happy Veteran’s Day, {name}. Thank you  for volunteering for service. What you’ve done is greatly appreciated. I know that this time can be difficult, if you need someone please reach out, we’re here for you if you need us.  -Riley"
```

Comment:
another `Riley` that is hard-coded perhaps? Should it be `{userName}`?

Reply:
Per above